### PR TITLE
86cuewz17/Styling 

### PR DESCRIPTION
**Pull request by nijin-26**
### Changes Made:

1. **Initialize Material-UI and Theming Configuration:**
   - Set up Material-UI in the project.
   - Configured the initial theming structure.
   - Add font sizes.

2. **Add Theme for Light and Dark Mode:**
   - Implemented themes for both light and dark modes using Material-UI.
   - Defined color schemes, typography, and other properties for each mode.

3. **Global Styles and Other Enhancements:**
   - Applied global styles to ensure a consistent look and feel.

### CODE REVIEW

**Comment by seraGeorge** [comment]
Make changes in the CHANGELOG.md

**Comment by nijin-26** [comment]
> Make changes in the CHANGELOG.md

I have updated it now.

**Review by seraGeorge — APPROVED** [review]

**Review by abhijith-biju — APPROVED** [review]

File: src/layout/Layout/Layout.tsx [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
 import { Footer } from '../Footer/Footer';
+import { Container } from '@mui/material';
 
 export const Layout = () => {
     return (
         <>
             <Header />
-            <main className="container">
-                <Outlet />
+            <main style={{ flexGrow: '1' }}>
```
Comment (amaljoeqb) [inline src/layout/Layout/Layout.tsx]: Can we separate styles to another file?

**Review by amaljoeqb — APPROVED** [review]

File: src/layout/Layout/Layout.tsx (hunk 1 above)
Comment (nijin-26) [reply src/layout/Layout/Layout.tsx]: Yes. I have updated it in the recent commit.

File: src/config/styles/globalStyles.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { GlobalStyles } from '@mui/material';
+
+export const GlobalStyle = () => (
+    <GlobalStyles
+        styles={() => ({
+            html: {
+                height: '100%',
+                fontSize: '10px',
```
Comment (lekshmi615213) [inline src/config/styles/globalStyles.tsx:8]: variable?

File: src/config/styles/theme.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                      secondary: '#9E9E9E',
+                  },
+              }
+            : {
+                  // palette values for dark mode
+                  primary: {
+                      main: '#E8DEF8',
+                  },
+                  secondary: {
+                      main: '#6750A4',
```
Comment (lekshmi615213) [inline src/config/styles/theme.ts]: variables

Comment (nijin-26) [reply src/config/styles/theme.ts]: I have changed the colors into constants in the recent commit.

**Pull request merged**